### DISCUSSION TITLE
Arbitrary page groupings

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1779] [Ideas]
It appears that a page can be part of only one collection at this time. However, it would be good if Outline could have arbitrary page groupings, to group pages in a "set" to flip through, instead of being fixed as part of a collection. 

The use case is something like a series of articles as in a set of blog posts, or, pages to digest for onboarding, or, sets of knowledge that might be grouped by "basic", "intermediate" or "advanced", for example. 

If pages are members of a "set", the set name appears at the top of the page, and has a way to navigate previous or next, through the set, across the boundaries of a more static collection. 

```
← NEW HIRE ONBOARDING 1 of 5 →
```

#### Edit 10 Jan 2021 - 

Thinking about this some more, yes, you could make an onboarding set in a collection, but since you cannot have docs homed in multiple collections, it kind of leads people to copy text out of document A and paste in document B. 

Another alternative to all this is just to make a document about onboarding, which links to all the documents you want a person to read in the onboarding process. This approach allows the writer to fill in any blanks for the target of the onboarding experience. Then, you can make a "Onboarding" collection and put the "index" docs in there.

**Comment by redbullpeter** [comment]
Having a document or sub document appear in multiple collections would be really useful to be able to curate content specific to an interest/function etc.
